Cargo.toml

```
@@ -1280,6 +1280,17 @@ description = "Bezier curve example showing a cube following a cubic curve"
 category = "Animation"
 wasm = true
 
+[[example]]
+name = "ease_functions"
+path = "examples/animation/ease_functions.rs"
+doc-scrape-examples = true
+
+[package.metadata.example.ease_functions]
+name = "Ease Functions"
+description = "Showcase of the built-in ease functions"
```

Nit: "Easing functions" might read better? The example also says uses it. Could consider renaming the file to match as well, but no strong opinion either way
```suggestion
description = "Showcases the built-in easing functions"
```